src/tool_getparam.c

```diff
@@ -1327,7 +1377,7 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
       bool raw_mode = (subletter == 'r');
 
       if(subletter == 'e') { /* --data-urlencode*/
-        /* [name]=[content], we encode the content part only
+        /* [name]=[content], we encode both name and content
```

Comment:
No! The docs for `--data-urlencode` states:
~~~
"name=content"
This will make curl URL-encode the content part and pass that on. Note that
the name part is expected to be URL-encoded already.

Reply:
You're right of course. I have reverted the name encoding.